api/boxes.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: page
+categories: ["API"]
+title: Boxes
+---
+
+# Boxes
+
+All API requests to `{{ site.parku.api }}/boxes` need a __private key__.
+
+## List all boxes
+
+```sh
+$ curl {{ site.parku.api }}/boxes \
+    -u 098f6bcd4621d373cade4e832627b4f6:parku
+```
+
+> Response
+
+```nginx
+Status: 200 OK
+```
+```json
+{
+  "services": ["FEFB"],
```

Comment:
A second service for the sake of understanding there might be _n_ services to scan for?